react/SelectBox/SelectBox.jsx

```diff
@@ -7,14 +7,16 @@ import { isIOSApp } from 'cozy-device-helper'
 
 import Icon from '../Icon'
 import CheckIcon from '../Icons/Check'
-import { dodgerBlue, silver, coolGrey, paleGrey } from '../palette'
+import palette from '../palette'
```

Comment:
@PolariTOON are you sure that this is not this line that create this big impact on the bundle size?

Reply:
We are already importing the palette in this way at other places and the file doesn't weight so much, thus i don't think so.